src/apps/loan-list/materials/selectable-material/selectable-material.tsx

```diff
@@ -88,7 +85,13 @@ const SelectableMaterial: FC<SelectableMaterialProps & MaterialProps> = ({
             {disabled && <CheckBox id={id} disabled={disabled} />}
           </div>
         )}
-        <div className="list-materials__content">
+        <div
+          className="list-materials__content cursor-pointer"
+          role="button"
```

Comment:
Previous comment regarding nested buttons also applies here and for other elements in this component:

> Appreciate the comment but consider dropping the `role="button"` attribute.

> This may cause nested button elements which is not allowed. Consequently we have *not* assigned roles to [elements in the card list component](https://github.com/reload/dpl-react/blob/cbb03875ea3cc1aa93bb9030dd54a8cbc1764abd/src/components/card-item-list/card-list-item/card-list-item.tsx#L108-L127).